components/Link/Link.js

```diff
@@ -49,7 +49,7 @@ class Link extends React.Component {
 
   render() {
     const { to, ...props } = this.props; // eslint-disable-line no-use-before-define
-    return <a href={typeof to === 'string' ? to : history.createHref(to)} {...props} onClick={this.handleClick} />;
+    return <a href={typeof to === 'string' ? to : history.createHref(to)} {...props} onClick={this.handleClick} />; // eslint-disable-line
```

Comment:
Using of `eslint-disable-line` is not obvious what exactly rule you want to ignore. It is better to always specify rule name in comment:
```js
let code; // eslint-disable-line rule-name
```
or
```js
// eslint-disable-next-line rule-name
let loooooongLineOfCode;
```

if eslint here warns about max-len, let's split the code into two lines:
```js
const href = typeof to === 'string' ? to : history.createHref(to);
return <a href={href} {...props} onClick={this.handleClick} />;
```